lib/scope.js

```diff
@@ -2286,7 +2289,7 @@ module.exports = {
     if ( error_handlers.system_error !== undefined ) {
       let sys_err_message = `The test tried to continue to the next page after ${ id }, but there was a system error. Check the picture of this page.`
       await scope.addToReport(scope, { type: `error`, value: sys_err_message });
-      expect( was_system_error, sys_err_message ).to.be.false;
+      expect( true, sys_err_message ).to.be.false;
```

Comment:
Was this meant to be changed back?

Reply:
Yes; `was_system_error` doesn't exist, and this change matches what we do when there is a user error. I noticed and changed it when I was trying to figure out what the error handler objects were and how they worked.

Reply:
```suggestion
      expect( was_system_error, sys_err_message ).to.be.false;
```

```suggestion
      expect( false, sys_err_message ).to.be.false;
```

```suggestion
```

Reply:
Or maybe

```suggestion
      expect( was_found, sys_err_message ).to.be.false;
```

Reply:
So it's meant to be kept the way it is, right?

Reply:
In which case, maybe we can figure out a better way than just true/false, which people seem to find confusing. But that doesn't have to happen this time around. Sorry I'm a little scattered at the moment.

Reply:
Sorry, I misread your question as "was this meant to be changed".  It was an intentional change. I don't think it needs to be changed back to the original.  I'm making code at least consistent with itself (the same `expect(` approach is used a few lines down).